exercises/practice/list-ops/.meta/list-ops.ys

```diff
@@ -19,10 +19,4 @@ defn foldl(list initial function):
   list.reduce(function initial)
 
 defn foldr(list initial function):
-  if len(list) > 0:
-    function _ list:first:
-      foldr: rest(list) initial function
-    else: initial
-
-defn reverse(list):
-  core/reverse: list
+  list:reverse.reduce(function initial)
```

Comment:
this is also not a good example of reimplementing the language in the language, though it already wasn't...